libraries/SITL/SIM_AirSim.h

```diff
@@ -60,14 +60,15 @@ class AirSim : public Aircraft {
 	bool parse_sensors(const char *json);
 
 	// buffer for parsing pose data in JSON format
-    uint8_t sensor_buffer[2048];
+    uint8_t sensor_buffer[65000];
```

Comment:
I had forgotten about this.
UDP packets have a size limitation of 65536 bytes, if we try to send data of bigger size, then the packet is not sent with an error of `Message Too Long`. After a bit of testing, happens around 45,000-50,000 points.

This could mean that we can limit the size of the packet rather than not have any packets at all. Currently, I'm using `asprintf` for allocating the string
https://github.com/microsoft/AirSim/pull/2075/files#diff-6bde6e8ea24246d8a48f2626f42cc99cR332

But rather than this, `snprintf` or something similar can be used

Reply:
Changed in Airsim PR
https://github.com/microsoft/AirSim/pull/2075/files#diff-6bde6e8ea24246d8a48f2626f42cc99cR332